### DISCUSSION TITLE
글  임시저장 기능

### DISCUSSION THREAD

**Discussion by liswktjs** [woowacourse-teams/2022-gong-seek#6] [Ideas]
크루들이 글을 작성할 때에, 
에디터에 작성중인 글을 임시저장 하는 기능을 제공할 것 인가?

**Comment by jurlring** [comment]
제공하면 좋을 것 같긴한데, 어떻게 구현하는게 좋을지 고민을 해봐야겠네요.
1. 사용자가 임시 저장 버튼을 누르면, 사용자마다 임시글 저장 테이블을 만들어서 저장한다. 

일단 이정도가 생각이 나는데, 관리 포인트가 하나 더 늘어나는 거라서 좀 생각이 필요할 것 같네요.
